vizro-core/hatch.toml

```diff
@@ -64,6 +64,7 @@ test-integration = [
   "pytest tests/integration --headless -k json {args}",
   "pytest tests/integration --headless -k yaml {args}"
 ]
+test-js = "./tools/run_jest.sh {args}"
```

Comment:
We should keep `test-js` script separate from the `test` script since `test-unit` and `test-integration` lie on the `pytest`. The reason is that the users can pass arguments like `hatch run test -vv` and `test-js`, that actually runs command `npx jest`, doesn't support `-vv` flag.